package/skelcd-control-suse-manager-server.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Wed Nov 27 17:26:55 UTC 2019 - Julio González Gil <jgonzalez@suse.com>
+
+- SUSE Manager version 4.2.0
```

Comment:
same as with proxy. You want to say 4.1.0

Reply:
Right! Fixed!